.env.test

```diff
@@ -4,4 +4,5 @@ CIRRUS_APP_ID=test_app_id
 CIRRUS_APP_NAME=test_app_name
 CIRRUS_CHANNEL=developer
 CIRRUS_FML_PATH=./tests/feature_manifest/sample.fml.yaml
-CIRRUS_SENTRY_DSN=
\ No newline at end of file
+CIRRUS_SENTRY_DSN=
+CIRRUS_INSTANCE_NAME=test_instance
```

Comment:
Missing newline at end of file.

You probably want to edit your editor configuration to automatically add newlines to files. If you are using VS Code, the setting is "Files: Insert Final Newline"

```
"files.insertFinalNewline": true
```